lib/rexml/parsers/xpathparser.rb

```diff
@@ -363,6 +363,18 @@ def Predicate path, parsed
         path
       end
 
+      def QuoteLiteral literal
```

Comment:
Could you use `snake_case`?

```suggestion
      def quote_literal(literal)
```

Reply:
And could you move this method to before `#LocalPath` (the line 192)?
Method in `#LocationPath` to `#FunctionCall` are for syntaxes in XPath. I don't want to mix this and them.